### PR TITLE
No negative zero

### PR DESCRIPTION
**Pull request by drewski3420**
The roundValue function returns a value of negative zero when the input (temperature) value is between -0.49 and -0.01 and roundTemp config value is set to true. 

Negative zero looks weird and incorrect when displayed in MM. My code returns 0 instead of -0 in those cases to resolve that issue. 

> Please send your pull requests the develop branch.
> Don't forget to add the change to CHANGELOG.md.

**Note**: Sometimes the development moves very fast. It is highly
recommended that you update your branch of `develop` before creating a
pull request to send us your changes. This makes everyone's lives
easier (including yours) and helps us out on the development team.
Thanks!

- Does the pull request solve a **related** issue?
- If so, can you reference the issue?
- What does the pull request accomplish? Use a list if needed.
- If it includes major visual changes please add screenshots.

### CODE REVIEW

File: modules/default/currentweather/currentweather.js [hunk 1]
```diff
@@ -587,6 +587,6 @@ Module.register("currentweather", {
 	 */
 	roundValue: function (temperature) {
```
Comment (rejas) [inline modules/default/currentweather/currentweather.js:588]: This function is used in 3 places. Could it be moved into some kind of utility class?

Comment (drewski3420) [reply modules/default/currentweather/currentweather.js:588]: I....don't know? I didn't see any obvious place to placfe functions that are shared across modules.

Comment (rejas) [reply modules/default/currentweather/currentweather.js:588]: Then nevermind, just some random thought of mine....

**Review by rejas — APPROVED** [review]
LGTM!

**Review by rejas — APPROVED** [review]

**Comment by khassel** [comment]
this PR is not based on `develop` (but on `master`) so this needs to be changed before merging.

**Comment by MichMich** [comment]
Could you please submit this PR to the `develop` branch?

**Comment by drewski3420** [comment]
> Could you please submit this PR to the `develop` branch?

Just switched to develop. Or at least I think I did. I'm new at this.

**Comment by sdetweil** [comment]
@drewski3420 

to submit code to MM as a pr, u need to 

create a personal copy (fork) of the MM repo (press the fork button, top right on mm code page)
then download that copy (git clone)
note i can only have one folder of the same name, so u may have to rename your old MM)

switch to develop
git checkout develop
npm install
make a new branch (using develop as the base) 
git checkout -b some_name
make your changes
git status shows what changed

git add files that changed u want to submit
git commit -m" some message about this commit of changes"

upload them to your fork 
git push (if new branch u will be shown a new command to replace this one, use it)
userid prompt
password prompt (next year have to use private codes from github)

switch to the web page for your fork, should see u submitted changes, click new pull request
make sure MM branch is marked as develop, and yours in you new branch name

fill in the comments, link to your issue #issue_number

create pr

**Comment by MichMich** [comment]
I understand what this does. But I had to read the code 3 times. There must be a cleaner way to solve this. 

What if we just use a variable to store the calculated value. And then to something like:

```
return roundValue !== -0 ? roundValue : 0
```

**Comment by sdetweil** [comment]
sure, I would do it the other way. if equal -0, as that is the problem child

**Comment by MichMich** [comment]
Yeah. Just updated my comment. 😆

**Comment by MichMich** [comment]
@drewski3420 is this a modification you can apply to your PR?

**Pull request merged**